src/main/java/com/nukkitx/blockstateupdater/BlockStateUpdater_1_16_0.java

```diff
@@ -107,6 +114,19 @@ public void registerUpdaters(CompoundTagUpdaterContext context) {
         this.addRequiredValueUpdater(context, "minecraft:melon_stem", "facing_direction", 0);
     }
 
+    private void addLegacyWallUpdater(CompoundTagUpdaterContext context, String name) {
+        context.addUpdater(1, 16, 0)
```

Comment:
These were never part of the block state updater in vanilla IIRC, so the version should be ignored.

Reply:
The `addUpdater()` doesn't have `ignoreVersion` flag set because we actually want to update the `"version"` inside of the block state to be 1.16.0.